lib/Db.php

```diff
@@ -16,20 +16,21 @@
 namespace Pimcore;
 
 use Pimcore\Db\Connection;
```

Comment:
```suggestion
use Doctrine\DBAL\Connection;
```
Here we could use `Doctrine\DBAL\Connection` already, as it would be redundant by adding `ConnectionInterface` right?

Reply:
ConnectionInterface does not have all the methods as Connection has. For BC reasons we need both here.

Reply:
But shouldn't we include `Doctrine\DBAL\Connection` in the `@return` as well, as it has additional methods which are not covered by `Pimcore\Db\ConnectionInterface` and `Pimcore\Db\Connection` ?

Reply:
@dpfaffenbauer have you noticed my above comment? 😊